### DISCUSSION TITLE
Prisma Relation Acts As List

### DISCUSSION THREAD

**Discussion by danwetherald** [prisma/prisma#5978] [Q&A]
Anyone have some cool methods / middleware to manage order lists in prisma via a position field?

Something like this from the Rails AR world comes to mind as an example: https://github.com/brendon/acts_as_list

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
